go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/stretchr/testify v1.9.0
 	github.com/temporalio/ui-server/v2 v2.29.2
 	go.temporal.io/api v1.36.0
-	go.temporal.io/sdk v1.27.0
+	go.temporal.io/sdk v1.28.2-0.20240816033637-2a02c483658a
```

Comment:
Upgraded sdk to point to the version where TQ stats were introduced in DescribeTaskQueueEnhanced (#1553)